src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta
+    name="viewport"
+    content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+    "
+  >
   <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-  <h1>Product cards</h1>
+  <div class="card" data-qa="card">
+    <div class="card__pic">
+      <img
+        src="images/imac.jpeg"
+        alt="Picture of product"
+        class="card__img"
+      >
+    </div>
+    <div class="description">
+      <div class="description__model">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="description__number">
+        Код товара: <span class="description__id">195434</span>
+      </div>
+    </div>
+    <div class="rate">
+      <div class="rate__stars stars--4">
+        <div class="rate__item"></div>
+        <div class="rate__item"></div>
+        <div class="rate__item"></div>
+        <div class="rate__item"></div>
+        <div class="rate__item"></div>
+      </div>
+      <div class="rate__review">
+        Отзывов: <span class="rate__rating">5</span>
+      </div>
+    </div>
+    <div class="price">
+      <div class="price__text">Цена:</div>
+      <div class="price__sum">
+        <span class="price__number">69 999</span> <span class="price__currancy">грн</span>
```

Comment:
Один тег - одна строка
```suggestion
        <span class="price__number">69 999</span>
        <span class="price__currency">грн</span>
```